html_permalinks_icon key error in GitHub pages builds

Since #112 it looks like the GitHub pages builds have been failing with a `html_permalinks_icon` key error (which means the last 4 tools to be merged (microfilm, tueplots, grplot and distinctipy) are not appearing on the docs page.

E.g. see this latest build: https://github.com/matplotlib/mpl-third-party/runs/7224799482?check_suite_focus=true

```
Exception occurred:
  File "/usr/local/lib/python3.8/site-packages/pydata_sphinx_theme/__init__.py", line 45, in update_config
    icon_default = app.config.values["html_permalinks_icon"]
KeyError: 'html_permalinks_icon'
The full traceback has been saved in /tmp/sphinx-err-kzajtp7a.log, if you want to report the issue to the developers.
Please also report this if it was a user error, so that a better error message can be provided next time.
A bug report can be filed in the tracker at <https://github.com/sphinx-doc/sphinx/issues>. Thanks!
make: *** [Makefile:21: html] Error 2
Traceback (most recent call last):
  File "/entrypoint.py", line 22, in <module>
    action.build_all_docs(github_env, [os.environ.get("INPUT_DOCS-FOLDER")])
  File "/sphinx_action/action.py", line 167, in build_all_docs
    raise RuntimeError("Build failed")
RuntimeError: Build failed
```

--- a/.github/workflows/gh-pages.yml
+++ b/.github/workflows/gh-pages.yml
@@ -14,10 +14,18 @@ jobs:
       - name: checkout
         uses: actions/checkout@v2
 
-      - name: Make docs builds
-        uses: ammaraskar/sphinx-action@master
+      - name: Install Python
+        uses: actions/setup-python@v4
         with:
-          docs-folder: "docs/"
+          python-version: '3.x'
+
+      - name: Install documentation requirements
+        run: |
+          pip install -r docs/requirements.txt
+
+      - name: Make docs builds
+        run: |
+          make -C docs html
 
       - name: Deploy
         uses: peaceiris/actions-gh-pages@v3

